kernelci.org/content/en/docs/org/tsc.md

```diff
@@ -41,3 +41,39 @@ used instead.
 
 The TSC also meet monthly online to discuss current topics and make decisions
 including votes when necessary.
+
+## Rules
+
+### Votes
+
+Most decisions are reached by consensus and don't require a vote.  Decisions
```

Comment:
> Most decisions are reached by consensus and don't require a vote.

I'm against this statement. 
We need to decide what require a vote and what doesn't require a vote in more detail. what if someone delete KernelCI GitHub ? will that need a vote or will be just "consensus"? 
KernelCI TSC should always need a vote and be completely transparent and recorded about discussion and decisions, if some decision can be accepted by consensus is not a topic of TSC but of maintainers or the weekly meeting. As quoted from the KernelCI TSC docs <The Technical Steering Committee (TSC) is a team of people who are responsible for keeping the KernelCI project in a good shape and driving its development.> for preserving the part of "team of people who are responsible" every TSC member need a vote on the topic discussed by TSC.

Reply:
If someone deletes a project on GitHub on purpose and without discussing it with anyone first, then it's obviously wrong and the person would most likely be removed from the TSC (with a vote...) if they're a member and have their admin permissions removed from GitHub.

Consensus for the TSC means the idea needs to be added to a meeting's agenda or on the kernelci-tsc mailing list, or at least on some chat channel e.g. Slack or IRC and others have a chance to say what they think.  It doesn't mean "free for all".  And as I've explained before already, if a consensus decision is made during a TSC meeting but one member could not attend and disagrees then it's always possible to restart the discussion.  We could have a rule that decisions should always have a date for when changes become effective, to leave some time for consideration and others to think about it.

Reply:
yes but you cannot know who voted for a decision. what if someone join a TSC meeting and maybe there are only three people and they decide to remove KernelCI, by consensus such thing is allowed and is even not tracked. I think that the added step about voting should help improve the transparency of the KernelCI TSC.
One example is how for example Gentoo manage technical discussion and decisions: https://projects.gentoo.org/council/meeting-logs/20220814.txt

Reply:
To be honest, if someone wanted to sabotage KernelCI they wouldn't even bother bringing it up as a discussion topic and there wouldn't be anything close to a consensus.  Or if they made it look like a consensus decision then it would obviously be addressed by the board and the offenders would most likely be removed from the TSC or have their admin rights removed etc. (as explained before)  So that's a fake issue, the real thing is we need to trust the people we're giving admin rights to with secure credentials (2FA) etc.  We also need recovery mechanisms in case some kind of disaster happens, voluntarily or not.

Reply:
That's not always so much obvious, like in the example I wrote. as that are only examples. Please take in consideration how other organizations are managing technical discussion and decisions.

Reply:
> the real thing is we need to trust the people we're giving admin rights to with secure credentials (2FA) etc. We also need recovery mechanisms in case some kind of disaster happens, voluntarily or not.

This is a issue but off topic in this pull request

Reply:
@aliceinwire I guess this is the rules followed by the Gentoo council?
https://wiki.gentoo.org/wiki/Project:Council